packages/syft/tests/syft/core/node/domain/domain_test.py

```diff
@@ -144,3 +144,14 @@ def test_object_exists_on_domain(domain: sy.Domain) -> None:
     assert not ptr.exists
     ptr = x.send(domain_client, id_at_location_override=uid)
     assert ptr.exists
+
+
+@pytest.mark.asyncio
+def test_create_user(domain: sy.Domain) -> None:
+    domain.create_user(email="test@test.com", password="changethis", budget=1)
```

Comment:
Could we replace the domain with `root_client` fixture as the create_user is present only on Client Object.

Reply:
@gmuraru  Ping, I think we get this test to wok, we are good for merge on the PR.